### PR TITLE
Enable dupword linter & fix findings

### PR DESCRIPTION
**Pull request by jtraglia**
**What type of PR is this?**

Documentation

**What does this PR do? Why is it needed?**

Removes all of the duplicate words which are typos.

### CODE REVIEW

File: .bazelversion [hunk 1]
```diff
@@ -1 +1 @@
-6.1.0
```
Comment (terencechain) [inline .bazelversion:1]: Is this related?

Comment (jtraglia) [reply .bazelversion]: Oops, nope that's an unintended change. Sorry about that. Changed it because I was using a newer version of bazel:

> ERROR: The project you're trying to build requires Bazel 6.1.0 (specified in /Users/jtraglia/Projects/jtraglia/prysm/.bazelversion), but it wasn't found in /opt/homebrew/Cellar/bazel/6.1.1/libexec/bin.

**Review by prestonvanloon — APPROVED** [review]
LGTM - Who knew there were so many violations of this? lol. nice work

**Review by rkapka — APPROVED** [review]
